frontend/src/Commons/Forms/InputModules/FarmInformation/FarmInformation.tsx

```diff
@@ -30,7 +30,6 @@ const FarmInfoComponent: React.FC<InputModuleProps> = ({ farmDetails, updateFarm
     Year: farmDetails.Year,
     FarmRegion: farmDetails.FarmRegion,
   };
-
   const validationSchema = Yup.object().shape({
     FarmName: Yup.string().max(24).required('Required'),
     Year: Yup.number().min(1900).max(2099).required('Required'),
```

Comment:
@GDamaso this is going to send me a type number instead of a string. There would be a type warning that only shows on dev mode that it's passing a number when expecting a string.

Although I can adjust my props validation and check for either number or string. It will leave a typescript error but will still be working.

Reply:
If you checkout to this branch, comment my props validation, you'll see an eslint error regarding props-validation.
If you proceed with the props-validation, make the type of all instances of year to be string, and submit complete data for farm information. Check console and you'll see a warning regarding expected string but got number

Reply:
Since I think Formik is validating for a number then it considers it as a number input field? That's why when we submit it's values. It's considering it as a type number and not a string.